projects/schematics/src/migrations/5_0/methods-and-properties-deprecations/data/methods-and-properties.migration.ts

```diff
@@ -0,0 +1,1790 @@
+      importPath: '@spartacus/cart/quick-order/root',
+      deprecatedNode: 'removeEntry',
+      comment:
+        "// TODO:Spartacus - Method 'removeEntry' was removed from Class 'QuickOrderFacade'. ",
+    },
+    {
+      class: 'QuickOrderFacade',
+      importPath: '@spartacus/cart/quick-order/root',
+      deprecatedNode: 'search',
+      comment:
+        "// TODO:Spartacus - Method 'search' was removed from Class 'QuickOrderFacade'. ",
+    },
+    {
+      class: 'AddToSavedCartComponent',
+      importPath: '@spartacus/cart/saved-cart/components',
+      deprecatedNode: 'activeCartService',
+      comment:
+        "// TODO:Spartacus - Property 'activeCartService' was removed from Class 'AddToSavedCartComponent'. ",
+    },
+    {
+      class: 'SavedCartAdapter',
+      importPath: '@spartacus/cart/saved-cart/core',
+      deprecatedNode: 'saveCart',
+      comment:
+        "// TODO:Spartacus - Method 'saveCart' was removed from Class 'SavedCartAdapter'. ",
+    },
+    {
+      class: 'SavedCartConnector',
+      importPath: '@spartacus/cart/saved-cart/core',
+      deprecatedNode: 'saveCart',
+      comment:
+        "// TODO:Spartacus - Method 'saveCart' was removed from Class 'SavedCartConnector'. ",
+    },
+    {
+      class: 'SavedCartEventBuilder',
+      importPath: '@spartacus/cart/saved-cart/core',
+      deprecatedNode: 'registerDeleteSavedCartEvents',
+      comment:
+        "// TODO:Spartacus - Method 'registerDeleteSavedCartEvents' was removed from Class 'SavedCartEventBuilder'. It was moved to 'CartEventBuilder', and was renamed to 'registerDeleteCart'.",
+    },
+    {
+      class: 'OccSavedCartAdapter',
+      importPath: '@spartacus/cart/saved-cart/occ',
+      deprecatedNode: 'getSaveCartEndpoint',
+      comment:
+        "// TODO:Spartacus - Method 'getSaveCartEndpoint' was removed from Class 'OccSavedCartAdapter'. ",
+    },
+    {
+      class: 'OccSavedCartAdapter',
+      importPath: '@spartacus/cart/saved-cart/occ',
+      deprecatedNode: 'saveCart',
+      comment:
+        "// TODO:Spartacus - Method 'saveCart' was removed from Class 'OccSavedCartAdapter'. ",
+    },
+    {
+      class: 'NewSavedCartOrderEntriesContext',
+      importPath: '@spartacus/cart/saved-cart/root',
+      deprecatedNode: 'actionsSubject',
+      comment:
+        "// TODO:Spartacus - Property 'actionsSubject' was removed from Class 'NewSavedCartOrderEntriesContext'. ",
+    },
+    {
+      class: 'SavedCartOrderEntriesContext',
+      importPath: '@spartacus/cart/saved-cart/root',
+      deprecatedNode: 'actionsSubject',
+      comment:
+        "// TODO:Spartacus - Property 'actionsSubject' was removed from Class 'SavedCartOrderEntriesContext'. ",
+    },
+    {
+      class: 'CartNotEmptyGuard',
+      importPath: '@spartacus/checkout/components',
+      deprecatedNode: 'activeCartService',
+      comment:
+        "// TODO:Spartacus - Property 'activeCartService' was removed from Class 'CartNotEmptyGuard'. ",
```

Comment:
related to this question about whether we should even include constructor properties deletion/renamed/added to methods-and-properties. https://github.com/SAP/spartacus/pull/15307/files#r830749914

This case is a good example too as this was just renamed to activeCartFacade, but again, I would say it's not important to know what the property named changed to